tests/negative/luks/creation.go

```diff
@@ -0,0 +1,58 @@
+// Copyright 2023 CoreOS, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package luks
+
+import (
+	"github.com/coreos/ignition/v2/tests/register"
+	"github.com/coreos/ignition/v2/tests/types"
+)
+
+func init() {
+	register.Register(register.NegativeTest, LuksOnNonExtDevice())
+}
+
+func LuksOnNonExtDevice() types.Test {
+	name := "tang.filesystem.luks.encrypt"
```

Comment:
So I am getting an error from this test (expected). Negative tests in my mind imply tests that should not pass due to various reasons. Currently its causing ci failure.

Reply:
Are blackbox negative tests actually passing scenarios? In that case I guess this test can be removed.. @jlebon wdyt?

Reply:
Possibly the test overall is failing because of calling `luksClose` on a non-existing device (see other comment).

Reply:
Yeah that makes sense. Updated thank you.